shelley-ma/shelley-ma-test/test/Tests.hs

```diff
@@ -15,17 +28,25 @@ maryChainExamples =
     [ multiAssetsExample
     ]
 
-tests :: TestTree
-tests =
+mainTests :: TestTree
+mainTests =
   testGroup
-    "Cardano-Legder-Tests"
+    "Allegra Legder Tests"
     [ Serialisation.tests,
       allegraTranslationTests,
       maryTranslationTests,
```

Comment:
Slightly confusing to have "mary" tests inside a block called "Allegra Ledger Tests"

Reply:
👍